### DISCUSSION TITLE
There is no notification class that can handle event `Spatie\Backup\Events\HealthyBackupWasFound`

### DISCUSSION THREAD

**Discussion by olipayne** [spatie/laravel-backup#1341] [Q&A]
Works fine in local, in production however I'm getting the following stack trace when running `php artisan backup:monitor`

```
[2021-06-29 12:12:10] production.ERROR: There is no notification class that can handle event
`Spatie\Backup\Events\HealthyBackupWasFound`. {"exception":"[object] 
Spatie\\Backup\\Exceptions\\NotificationCouldNotBeSent(code: 0):
There is no notification class that can handle event `Spatie\\Backup\\Events\\HealthyBackupWasFound`. 
at /home/forge/{site}/vendor/spatie/laravel-backup/src/Exceptions/NotificationCouldNotBeSent.php:13)
[stacktrace]
```

**Comment by olipayne** [comment]
Hello! Not sure of the reason this was moved from an issue to a discussion?

**Reply by freekmurze** [reply]
Because it’s likely a configuration problem at your end and not a bug in the package code.

**Reply by olipayne** [reply]
Ok, it's bizarre that the exact same config works in local vs my production environment, any pointers on where I can start digging?

Thanks!

**Reply by skyrpex** [reply]
I had the same problem. Turns out I upgraded the laravel-backup package but not quite the config file, and the notification class names are different now (they now have a "Notification" suffix). That's why it said that there's no notification class that can handle the events.

**Reply by PaolaRuby** [reply]
>All notification class names got suffixed with 'Notification'. Make sure you update the classnames where you are using these notifications.

[UPGRADING.md](https://github.com/spatie/laravel-backup/blob/main/UPGRADING.md)
